NFIQ2/NFIQ2Algorithm/src/nfiq2/nfiq2_qualityfeatures_impl.cpp

```diff
@@ -181,397 +188,63 @@ NFIQ::QualityFeatures::Impl::getActionableQualityFeedback(
 						// (foreground)
 			fb_roi.identifier = NFIQ::
 			    ActionableQualityFeedbackIdentifier_SufficientFingerprintForeground;
-			actionableQuality.push_back(fb_roi);
+			actionableMap["SufficientFingerprintForeground"] =
+			    fb_roi;
 		}
 	}
 
+	actionableQuality.push_back(actionableMap.at("UniformImage"));
```

Comment:
Any reason we don't just return `actionableMap` instead of copying it into the vector?